tests/integration/update_cluster/externalpolicies/in-v1alpha2.yaml

```diff
@@ -0,0 +1,103 @@
+apiVersion: kops.k8s.io/v1alpha2
+kind: Cluster
+metadata:
+  creationTimestamp: "2016-12-10T22:42:27Z"
+  name: complex.example.com
+spec:
+  api:
+    loadBalancer:
+      type: Public
+      additionalSecurityGroups:
+      - sg-exampleid3
+      - sg-exampleid4
+  kubernetesApiAccess:
+  - 0.0.0.0/0
+  channel: stable
+  cloudProvider: aws
+  cloudLabels:
+    Owner: John Doe
+    foo/bar: fib+baz
+  configBase: memfs://clusters.example.com/complex.example.com
+  etcdClusters:
+  - etcdMembers:
+    - instanceGroup: master-us-test-1a
+      name: us-test-1a
+    name: main
+  - etcdMembers:
+    - instanceGroup: master-us-test-1a
+      name: us-test-1a
+    name: events
+  kubeAPIServer:
+    serviceNodePortRange: 28000-32767
+    auditWebhookBatchThrottleQps: 3.14
+  kubernetesVersion: v1.14.0
+  masterInternalName: api.internal.complex.example.com
+  masterPublicName: api.complex.example.com
+  networkCIDR: 172.20.0.0/16
+  networking:
+    kubenet: {}
+  nodePortAccess:
+  - 1.2.3.4/32
+  - 10.20.30.0/24
+  nonMasqueradeCIDR: 100.64.0.0/10
+  sshAccess:
+  - 0.0.0.0/0
+  topology:
+    masters: public
+    nodes: public
+  externalPolicies:
+    node:
+      - aws:arn:iam:123456789000:policy:test-policy
+    master:
+      - aws:arn:iam:123456789000:policy:test-policy
+    bastion:
+      - aws:arn:iam:123456789000:policy:test-policy
```

Comment:
I dont see this policy arn referenced in the terraform output at all.

I'd expect to see something like:

```tf
resource "aws_iam_role_policy_attachment" "node-externalpolicy-test-policy" {
  role       = "${aws_iam_role.nodes-complex-example-com.name}"
  policy_arn = "aws:arn:iam:123456789000:policy:test-policy"
}
```

somewhere in kubernetes.tf below.

Reply:
ah, I misunderstood how this worked. I'll fix that.